### PR TITLE
Update InputGroup

### PR DESCRIPTION
**Pull request by florettewest**

### CODE REVIEW

File: components/input-group/src/InputGroup.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * Input Group: Styled input fields with attached addons. Addons can be text ($, %, .00 etc) or form controls (buttons or select inputs).
+ */
+export const InputGroup = ({ size, children, ...props }) => {
+	const formContext = useFormContext();
+
+	const inputGroupSize = (formContext && formContext.size) || size;
+
+	const childrenWithProps = Children.map(children, child =>
+		cloneElement(child, { size: inputGroupSize })
+	);
```
Comment (jaortiz) [inline components/input-group/src/InputGroup.js]: Need to look into doing this via context. Would probably involve creating wrappers around the other gel components this is consuming. Leave for now and we can discuss with TM later.

File: components/input-group/src/InputGroupAddon.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		},
+	};
+
+	return (
+		<span
+			css={{
+				fontSize: sizeMap[size].fontSize,
+				lineHeight: 1.5,
+				padding: sizeMap[size].padding,
+				height: sizeMap[size].height,
```
Comment (jaortiz) [inline components/input-group/src/InputGroupAddon.js]: can change to 
```
...sizeMap[size]
```

File: components/input-group/src/InputGroupAddon.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+// ==============================
+// Types
+// ==============================
+
+InputGroupAddon.propTypes = {
+	/**  Component text */
+	children: PropTypes.string.isRequired,
+};
+
+InputGroupAddon.defaultProps = {};
```
Comment (jaortiz) [inline components/input-group/src/InputGroupAddon.js]: Remove since no default props

File: components/input-group/src/InputGroup.js (hunk 1 above)
Comment (jonnystening) [reply components/input-group/src/InputGroup.js]: Hey @florettewest this was already done in the previous PR you closed

File: components/input-group/src/InputGroupAddon.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+				backgroundColor: COLORS.light,
+				border: `1px solid ${COLORS.borderDark}`,
+				borderRadius: '0.1875rem',
+				borderTopRightRadius: 0,
+				borderBottomRightRadius: 0,
+				whiteSpace: 'nowrap',
+
+				borderRight: first && 0,
+				...(last && {
+					borderTopRightRadius: '3px',
```
Comment (jonnystening) [inline components/input-group/src/InputGroupAddon.js]: These should be in rems

**Pull request merged**